os/sys/etimer.h

```diff
@@ -189,12 +189,12 @@ clock_time_t etimer_start_time(struct etimer *et);
 /**
  * \brief      Check if an event timer has expired.
  * \param et   A pointer to the event timer
- * \return     Non-zero if the timer has expired, zero otherwise.
+ * \return     True if the timer has expired.
  *
  *             This function tests if an event timer has expired and
  *             returns true or false depending on its status.
  */
-int etimer_expired(struct etimer *et);
+bool etimer_expired(struct etimer *et);
```

Comment:
Missing `#include <stdbool.h>` here and in the other files?

Reply:
Fixed.